docs/ldml/tr35-info.md

```diff
@@ -974,10 +974,10 @@ There are many possible ways to construct complex units. For comparison of unit
 3. Convert multiple instances of a unit into the appropriate power.
    * foot-per-second-second ⇒ foot-per-square-second
    * kilogram-meter-kilogram ⇒ meter-square-kilogram
-4. For each single unit, disregarding prefixes and powers, find its base unit using `<convertUnit>`, then get the order of that base unit among the `unitQuantity` elements in the [units.xml](https://github.com/unicode-org/cldr/blob/master/common/supplemental/units.xml). Then sort the single units by that order.
+4. For each single unit, disregarding prefixes and powers, get the order of the _simple_ unit among the `unitQuantity` elements in the [units.xml](https://github.com/unicode-org/cldr/blob/master/common/supplemental/units.xml). Sort the single units by that order, using a stable sort. If there are private-use single units, sort them after all the non-private use single units.
    * meter-square-kilogram => square-kilogram-meter
    * meter-square-gram ⇒ square-gram-meter
-5. If two single units have the same simple unit but different SI prefixes, such as "kilometer-meter", sort the higher-power SI prefixes first.
+5. As an edge case, there could be two adjacent single units with the same _simple_ unit but different SI prefixes, such as _meter-kilometer_. In that case, sort the higher-power SI prefixes first, such as _kilometer-meter_.
```

Comment:
Do we want to specify binary prefixes too?

Currently ICU is sorting binary prefixes before SI prefixes - but then we're also sorting from small to large, so I need to change the order anyway. If I just flip the sign, it will be SI prefixes before binary prefixes.

Current test case demonstrating ICU sort order:

input: "megafoot-mebifoot-kibifoot-kilofoot"
output: "kibifoot-mebifoot-kilofoot-megafoot"